README_shelltest.md

```diff
@@ -0,0 +1,36 @@
+# CLI Testing
+The spellcheck team is testing the command line inputs using the ShelltestRunner program.
+
+The program takes in input, expected output, expected stderr, and expected stdout.
+It is important to note that the program is very precise in the sense that it even counts all the white spaces in between the lines. Therefore, it is really important to make sure all the white spaces are included in the expected outcomes, otherwise the tests fail.
+
+## How to download the program
+
+Instructions on downloading the program and its further use can be found in the following link: https://github.com/simonmichael/shelltestrunner
+
+For our purposes, the information that I outlined below should be enough. You can download the program by downloading stack(https://haskell-lang.org/get-started) first, and then you need to run the following in your terminal:
+
+stack install shelltestrunner-1.9
+
+## How to run the tests
```

Comment:
Can you also add information on how to generate the cli_batch and cli_interactive files, since you expect future PRs to pass the tests here as well?

Reply:
Done, 
See commit e0abec544499a99212526e0b1934d1f5fe52289e